From f0864779476b9135535908bd005e26ce57b638da Mon Sep 17 00:00:00 2001
From: Megan Wilhite <mwilhite@vmware.com>
Date: Fri, 25 Mar 2022 11:14:01 -0600
Subject: [PATCH] Fix requested feedback

(cherry picked from commit bb610f761e107de111fa8b4c92c60c77ff7a920e)

Rebased to salt 3000.2. Removed ci/ and pytest files from the commit
which do not exist in 3000.2.

Signed-off-by: Alex Stewart <alex.stewart@ni.com>

Upstream-Status: Inappropriate [Upstream stable EOL.]
---
 salt/utils/jinja.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/salt/utils/jinja.py b/salt/utils/jinja.py
index d2b91fb5a4..0861ce2a2d 100644
--- a/salt/utils/jinja.py
+++ b/salt/utils/jinja.py
@@ -44,10 +44,10 @@ except ImportError:
 
 
 try:
-    from jinja2 import Markup
-except ImportError:
-    # Markup moved to markupsafe in jinja>= 3.1
     from markupsafe import Markup
+except ImportError:
+    # jinja < 3.1
+    from jinja2 import Markup
 
 log = logging.getLogger(__name__)
 
@@ -665,9 +665,9 @@ def symmetric_difference(lst1, lst2):
     return unique([ele for ele in union(lst1, lst2) if ele not in intersect(lst1, lst2)])
 
 
-if jinja2.__version__ < "3.0.0":
+try:
     contextfunction = jinja2.contextfunction
-else:
+except AttributeError:
     contextfunction = jinja2.pass_context
 
 
